docs/nodes/build/launch-an-avalanche-validator-on-aws-with-one-click.md

```diff
@@ -0,0 +1,347 @@
+  action as "Launch from Website." The EC2 Instance Type should remain
+`c5.2xlarge`. The primary change you'll need to make is to choose a keypair
+which will enable you to `ssh` into the newly created EC2 instance to run `curl`
+commands on the Validator node. You can search for existing Keypairs or you can
+create a new keypair and download it to your local machine. If you create a new
+keypair you'll need to move the keypair to the appropriate location, change the
+permissions and add it to the OpenSSH authentication agent. For example, on
+MacOS it would look similar to the following: 
+
+```zsh
+# In this example we have a keypair called avalanche.pem which was downloaded from AWS to ~/Downloads/avalanche.pem
+# Confirm the file exists with the following command
+test -f ~/Downloads/avalanche.pem && echo "avalance.pem exists."
+
+# Running the above command will output the following:
+# avalance.pem exists.
+
+# Move the avalanche.pem keypair from the ~/Downloads directory to the hidden ~/.ssh directory
+mv ~/Downloads/avalanche.pem ~/.ssh
+  
+# Next add the private key identity to the OpenSSH authentication agent
+ssh-add ~/.ssh/avalanche.pem; 
+
+# Change file modes or Access Control Lists
+sudo chmod 600 ~/.ssh/avalanche.pem
+```
+
+Once these steps are complete you are ready to launch the Validator node on EC2.
+To make that happen click the "Launch" button 
+
+![launch successful](/img/one-click-validator-node/launch-successful.png)
+
+You now have an Avalanche Validator node deployed on an AWS EC2 instance! Copy
+the `AMI ID` and click on the `EC2 Console` link for the next step.
+
+## EC2 Console
+
+Now take the `AMI ID`  from the previous step and input it into the search bar
+on the EC2 Console. This will bring you to the dashboard where you can find the
+EC2 instances public IP address. 
+
+![AMI instance](/img/one-click-validator-node/ami-instance.png)
+
+Copy that public IP address and open a Terminal or command line prompt. Once you
+have the new Terminal open `ssh` into the EC2 instance with the following
+command. 
+
+```zsh
+ssh username@ip.address.of.ec2.instance
+```
+
+Now that you are `ssh`ed into the EC2 instance you can run the
+`info.isBoostrapped` command to confirm if the Avalanche Validator node has
+finished bootstrapping.
+
+### `info.isBootstrapped` Request
+
+```zsh
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"info.isBootstrapped",
+    "params": {
+        "chain":"X"
+    }
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/info
+```
+
+Once the node is finished bootstrapping, the response will be:
+
+### `info.isBootstrapped` Response
+
+```zsh
+{
+    "jsonrpc": "2.0",
+    "result": {
+        "isBootstrapped": true
+    },
+    "id": 1
+}
+```
+
+**Note** that initially the response is `false`. This is because it takes time
+for the node to sync the Avalanche Fuji testnet. When you are adding your node
```

Comment:
How do we know it is on Fuji not on Mainnet? Is it default to Fuji or Mainnet? Where is the configuration?

Reply:
Resolved